### ISSUE TITLE
403 errors in console while running E2E tests

### ISSUE THREAD

**Issue by ivatautomattic** [woocommerce/woocommerce#27700]
**Desc**:  When running[ e2e tests ](https://github.com/woocommerce/woocommerce/tree/master/tests/e2e) I get console errors 

**STR**:
1. Setup local env for E2E tests using the guide [here](https://github.com/woocommerce/woocommerce/tree/master/tests/e2e#prep-work-for-running-tests)
2. Run the tests in headless mode -  `npm run test:e2e`

**Related to**: #26724 

![image](https://user-images.githubusercontent.com/70089943/93486911-90312700-f8b9-11ea-8668-a70d461f993c.png)

**LOGS**: 
[2020-09-17T16_18_42_837Z-debug.log.txt](https://github.com/woocommerce/woocommerce/files/5240584/2020-09-17T16_18_42_837Z-debug.log.txt)

**UPDATE1**: We are having the same errors in Travis

**Comment by rrennick** [comment]
@ivatautomattic You get these from the tracking pixel if your proxy isn't active.

**Comment by ivatautomattic** [comment]
> @ivatautomattic You get these from the tracking pixel if your proxy isn't active.

Below is the error I get if I run the tests with my proxy not active. It is different from the 403 above. Have you had a chance to check the logs I attached?

```console.error
    Failed to load resource: net::ERR_PROXY_CONNECTION_FAILED

      174 | 
      175 | 		// eslint-disable-next-line no-console
    > 176 | 		console[ logFunction ]( text );
          | 		^
      177 | 	} );
      178 | }
      179 | 

      at Page.<anonymous> (src/setup/jest.setup.js:176:3)
      at Page._onLogEntryAdded (node_modules/puppeteer/lib/Page.js:224:12)
      at CDPSession.<anonymous> (node_modules/puppeteer/lib/Page.js:124:47)
      at CDPSession._onMessage (node_modules/puppeteer/lib/Connection.js:200:12)
      at Connection._onMessage (node_modules/puppeteer/lib/Connection.js:112:17)
```

**Comment by rrennick** [comment]
@ivatautomattic This was addressed in #27785 . In 4.6 we removed the `wc-setup` page. In 4.5 and earlier the `wc-setup` page launched onboarding.
